tests/containers/worker.pm

```diff
@@ -0,0 +1,19 @@
+use Mojo::Base qw(openQAcoretest);
+use testapi;
+use utils;
+
+sub run {
+  my ($self) = @_;
+  my $volumes = "-v \"/root/data/factory:/data/factory\" -v \"/root/data/tests:/data/tests\" -v \"/root/openQA/container/webui/conf:/data/conf:ro\"";
```

Comment:
Please use single-ticks by default unless you want to expand perl variables within. Then you would not need the additional quote escaping.

Reply:
Done

Reply:
well, it was meant as a generic suggestion, not just for this line :)

Please do the same for the other statements as well.

Reply:
Ok, done

Reply:
please also change https://github.com/os-autoinst/os-autoinst-distri-openQA/pull/61/files#diff-fba7d5b35a7ae1e16e0a7fd11bbcf260d1aa87d9323dd92eaaf0e300a5a799e5R10